.dockerignore

```diff
@@ -1,9 +1,16 @@
-.vagrant/
-_build/
-apps/aecuckoo/c_src
-apps/aecuckoo/priv/bin
-apps/aecuckoo/priv/lib
-apps/aecore/priv/keys
-system_test
-erl_crash.dump
-aevm_external
+*
+!rebar.config
+!rebar.lock
+!Makefile
+!rebar3
+!rebar.config.script
+!VERSION
+!apps
+!.git
+!otp_patches
+!test
```

Comment:
Can you please remind me why this needed?

Reply:
@lucafavatella ```                             {tag,"2.8.0"}}) as an app of the same name has already been fetched
===> Verifying dependencies...
make[1]: Entering directory '/app/otp_patches'
./compile.escript
Patches = ["20.1/mnesia.erl.patch","20.1/mnesia_index.erl.patch"]
make[1]: Leaving directory '/app/otp_patches'
===> Skipping jsx (from {git,"git://github.com/talentdeficit/jsx.git",
                             {tag,"2.8.0"}}) as an app of the same name has already been fetched
===> Skipping jsx (from {git,"git://github.com/talentdeficit/jsx.git",
                             {tag,"2.8.0"}}) as an app of the same name has already been fetched
===> Generating code from "config/swagger.yaml" writing to "apps/aeutils/src/endpoints.erl"
test/ct_eunit_xform.erl: no such file or directory
===> Hook for compile failed!

make: *** [internal-build] Error 1
Makefile:304: recipe for target 'internal-build' failed
The command '/bin/sh -c cd /app && make prod-build' returned a non-zero code: 2```

Reply:
I am attempting to remove this need by (WIP) PR https://github.com/aeternity/epoch/pull/1215

Reply:
An improvement may be including only `test/ct_eunit_xform.erl` (did not test).

Reply:
it is working with only test/ct_eunit_xform.erl